### ISSUE TITLE
Add a way to queue goals

### ISSUE THREAD

**Issue by leijurv** [cabaletta/baritone#176]
e.g. "baritone please don't be smart and try and go diagonal. please just go from 0,0 to 25000,0 to 25000,2000 to 24500,2000 to the portal there" btw those aren't my real base coords i promise

**Comment by plootia** [comment]
can confirm, don't bother going to those coords

**Comment by ThatGrzegorzB** [comment]
please yes

**Comment by ZeroMemes** [comment]
I assume the next goal in the queue would only be set if the current one is successfully reached, and if the current one fails all succeeding ones are dropped?

**Comment by ZeroMemes** [comment]
Bumping this, it is something that I’m actually capable of doing

**Comment by VolantTheFox** [comment]
bump! any news on this? it would be hella useful to go to a certain set of coords, then #mine <block> <amount>, and THEN go back to home base.

**Comment by mrphax0r** [comment]
Would it take much more than a list tho?

**Comment by GunnerBasil** [comment]
Bump. I would love this. like mine 3 diamonds, and make a pickaxe or something

**Comment by dominikaaaa** [comment]
don't comment useless shit for no reason, unless you have something to contribute to the conversation, as it sends a notification. this isn't a forum

**Comment by SylveonBottle** [comment]
Duplicate of #269, [according to l1ving](https://github.com/cabaletta/baritone/issues/269#issuecomment-751557525)

**Comment by SylveonBottle** [comment]
I found this issue personally resolved by using [AltoClef](https://github.com/gaucho-matrero/altoclef).

More specifically, [this fork of altoclef](https://github.com/Meloweh/altoclef) which allows running chain commands.

Just incase anyone from the mentioned issues would like a temporary solution / something to toy with.

**Comment by walksanatora** [comment]
yeah but we shouldn't have to use forks/addons to use a feature that should be in the base mod

**Comment by SiffrinSuffrin** [comment]
Would also be cool to automatically set up bounding boxes, ie
#queue goto 0 0 0
#queue sel pos1
#queue goto 100 0 100
#queue sel pos2
#queue sel cleararea
#queue home

**Comment by bp88dev** [comment]
Seems like this suggestion has lost its momentum, but I think this would be a great addition to Baritone.

**Comment by ghost** [comment]
waiting for this solid 7years!!!
